app/containers/MessageBox/index.js

```diff
@@ -352,60 +353,33 @@ class MessageBox extends Component {
 	}
 
 	// eslint-disable-next-line react/sort-comp
-	debouncedOnChangeText = debounce(async(text) => {
-		const { sharing } = this.props;
-		const db = database.active;
+	debouncedOnChangeText = debounce((text) => {
 		const isTextEmpty = text.length === 0;
-		// this.setShowSend(!isTextEmpty);
-		this.handleTyping(!isTextEmpty);
-
-		if (!sharing) {
-			// matches if their is text that stats with '/' and group the command and params so we can use it "/command params"
-			const slashCommand = text.match(/^\/([a-z0-9._-]+) (.+)/im);
-			if (slashCommand) {
-				const [, name, params] = slashCommand;
-				const commandsCollection = db.get('slash_commands');
-				try {
-					const command = await commandsCollection.find(name);
-					if (command.providesPreview) {
-						return this.setCommandPreview(command, name, params);
-					}
-				} catch (e) {
-					console.log('Slash command not found');
-				}
-			}
+		if (isTextEmpty) {
+			this.stopTrackingMention();
+			return;
 		}
+		this.handleTyping(!isTextEmpty);
 
-		if (!isTextEmpty) {
-			try {
-				const { start, end } = this.selection;
-				const cursor = Math.max(start, end);
-				const lastNativeText = this.text;
-				// matches if text either starts with '/' or have (@,#,:) then it groups whatever comes next of mention type
-				let regexp = /(#|@|:|^\/)([a-z0-9._-]+)$/im;
-
-				// if sharing, track #|@|:
-				if (sharing) {
-					regexp = /(#|@|:)([a-z0-9._-]+)$/im;
-				}
-
-				const result = lastNativeText.substr(0, cursor).match(regexp);
-				if (!result) {
-					if (!sharing) {
```

Comment:
As wrote on this condition, slash commands shouldn't be triggered when `sharing === true`.
You can reproduce it by sharing images and then starting a message with `/`.